### ISSUE TITLE
Thanks! 

### ISSUE THREAD

**Issue by mithi** [mauriciopoppe/blog#2]
I'm currently building a simple hexapod robot movement simulator from scratch 
and it's so much harder than I anticipated. 

Your notes really helped me a lot. 
The choice of font is great, the animations and diagrams are so helpful. 

Of course, the text explains the concepts very well as well. 

I just wanted to say thank you. 

I'm just getting started but might be interested in checking out the project here.
https://github.com/mithi/hexapod-robot-simulator

Thanks again!

**Comment by mauriciopoppe** [comment]
Hey Mithi,

Im glad it helped you, you’ve got some awesome projects too, I’ll follow
your projects too since I’m actually doing a MSCS degree in robotics.

Feel free to ping me if you find something fun to look at or if you have
questions about what I wrote.

Mauricio

**Comment by mithi** [comment]
Hi @mauriciopoppe ,

Just wanted to share that I have finished the bare minimum features of my project which I shared with you last February. 

https://github.com/mithi/hexapod-robot-simulator

It's still has a few issues ((most important #41 #33 #1) that I'm not really sure how to fix as of now) but it's already  usable. I know you're busy, but I'm just writing in case you have a couple of seconds to check my project out.

Thanks again!
